### DISCUSSION TITLE
Auto update?

### DISCUSSION THREAD

**Discussion by b-u-g-s** [Pithus/bazaar#22] [Q&A] [answered]
Hi again,

I am just wondering how the docker get updated once you have created the bazaar app?
Do I need to update the git clone then re-run the docker compose?
Or once created the docker bazaar app will auto update? (beyond the tools databases, I mean updating with any changes linked to your git repo)

I am just keen of keeping the latest version and fixes coming :)
B.

**Comment by U039b** [comment, marked as answer]
For the moment, Pithus is in beta so there is no automatic Docker images release. In the future, we plan to deliver updates through Docker Hub. A service such as [Watch Tower](https://github.com/containrrr/watchtower) would ensure automatic stack update.

**Reply by b-u-g-s** [reply]
Ok, so if I run a gitupdate and then a docker compose --force-recreate would it be enough to update the images?
Or we are speaking removing the bazaar images from docker and a run again a "docker compose built"?

**Reply by U039b** [reply]
Git update then `docker-compose build` and finally `docker-compose up` would do the job :)

**Reply by b-u-g-s** [reply]
great thanks!
